### ISSUE TITLE
Ari.client.sounds.list results in NameError

### ISSUE THREAD

**Issue by mdunc** [svoboda-jan/asterisk-ari#2]
model.rb seems to get confused about having the parameters for sounds in an options object.

```
?> Ari.client.sounds.list
NameError: `@{"language"=>"en", "format"=>"gsm"}' is not allowed as an instance variable name
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:20:in `instance_variable_set'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:20:in `block in attributes='
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:14:in `each'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:14:in `attributes='
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:9:in `initialize'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/resources/sound.rb:18:in `new'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/resources/sound.rb:18:in `formats='
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:18:in `block in attributes='
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:14:in `each'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:14:in `attributes='
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/model.rb:9:in `initialize'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/resources/sound.rb:35:in `new'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/resources/sound.rb:35:in `block in list'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/resources/sound.rb:35:in `map'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/resources/sound.rb:35:in `list'
	from /usr/local/bundle/gems/asterisk-ari-client-0.0.7/lib/ari/list_resource.rb:16:in `method_missing'
```

**Comment by svoboda-jan** [comment]
Yeah, it wasn't handling list properties. Just released version `0.0.8` with a fix.

**Comment by mdunc** [comment]
Awesome!  Thank you!  This is an awesome library you've written by the way.
